addons/mail/static/tests/chatter/web/follower.test.js

```diff
@@ -186,9 +186,9 @@ test("removing a follower should reload form view", async function () {
         res_id: threadId,
         res_model: "res.partner",
     });
-    onRpc((route, args) => {
-        if (route === "/web/dataset/call_kw/res.partner/web_read") {
-            step(`read ${args.args[0][0]}`);
+    onRpc("web_read", ({ args, model }) => {
+        if (model === "res.partner") {
```

Comment:
Hello, would it be possible to have a better syntax for ORM methods?

`onOrm(modelName, methodName, ...)` or something like that?

It makes more sense to see the model first, and it's more convenient than catching all methods that have the same name and having to filter on model later.